modules/services/emacs.nix

```diff
@@ -81,6 +81,14 @@ in {
       enable = mkEnableOption "systemd socket activation for the Emacs service";
     };
 
+    startWithUserSession = lib.mkOption {
+      default = !cfg.socketActivation.enable;
```

Comment:
Need to add something like

    defaultText = literalExample "config.services.emacs.socketActivation.enable";

otherwise the manual will depend on the configuration, wihch causes unnecessary rebuilds.

Reply:
fixed. I don't understand what I'm doing but I replaced `literalExample` with `literalExpression`.

Reply:
Sorry, force of habit. The `literalExpression` is correct.